packages/knip/src/plugins/astro/index.ts

```diff
@@ -41,7 +48,6 @@ export default {
   enablers,
   isEnabled,
   config,
-  entry,
-  production,
```

Comment:
Sorry to bother you once more, but could you please restore and define `entry` and `production` with default values?

That's for the documentation generator to [read default values](https://github.com/webpro-nl/knip/blob/1bccd592a6cd4652c6dd755c276980332113fc8f/packages/docs/scripts/generate-plugin-docs.ts#L61-L63): https://knip.dev/reference/plugins/astro#default-configuration (`entry` and `production` will be merged and shown as `entry` here).

If `resolveConfig` or `resolveAST` is implemented, it will be ignored: https://github.com/webpro-nl/knip/blob/1bccd592a6cd4652c6dd755c276980332113fc8f/packages/knip/src/WorkspaceWorker.ts#L344

In `resolveAST` we can then read `entry` and `production` and replace the default `srcDir` with the return value of `getSrcDir()`.

Reply:
Done